src/CourierClient.php

```diff
@@ -9,12 +9,46 @@
 use Psr\Http\Message\ResponseInterface;
 use Shuttle\Shuttle;
 
-final class CourierClient
+interface CourierClientInterface {
```

Comment:
Usually, we would be extracted to its own file.

Reply:
great suggestion!